x-pack/plugins/observability_solution/dataset_quality/public/components/dataset_quality_details/degraded_field_flyout/possible_mitigations/field_limit/field_limit_documentation_link.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiLink } from '@elastic/eui';
+import { useKibanaContextForPlugin } from '../../../../../utils';
+import { fieldLimitMitigationOfficialDocumentation } from '../../../../../../common/translations';
+
+export function FieldLimitDocLink() {
+  const {
+    services: { docLinks },
+  } = useKibanaContextForPlugin();
+
+  return (
+    <EuiLink
+      data-test-subj="datasetQualityManualMitigationsPipelineLink"
+      href={docLinks.links.elasticsearch.mappingSettingsLimit}
+      external
+      target="_blank"
```

Comment:
> [!NOTE]
> Either `external` or `target="_blank"` are sufficient to make the link external as per EUI documentation, maybe we can remove the `target` attribute here.

Reply:
I think there is some sort of bug in EUI Documentation.
You need the `target="_blank" always. Else it won't open in a new tab

That external infact is redundant. Not sure why we have it. May be they are in process of deprecating it